wagtail/images/static_src/wagtailimages/js/add-multiple.js

```diff
@@ -205,7 +205,11 @@ $(function () {
     }).done(function (data) {
       if (data.success) {
         var statusText = $('.status-msg.update-success').text();
-        addMessage('success', statusText);
+        document.dispatchEvent(
+          new CustomEvent('w-messages:add', {
+            detail: { status: 'success', text: statusText },
```

Comment:
```suggestion
            detail: { status: 'success', text },
```

If you rename the variable above to `text`, you can set the key/value the same with this shortcut here.

Also, maybe 'text' is not the right variable name, it might be worth considering `content` or even `message`, if you find that you are writing code to rename variables for readability (i.e. in the controller you are renaming text to statusText), maybe the name needs a revisit.

Reply:
how about `messageContent` or `messageText` ?

Reply:
Maybe let's keep as text